src/index.js

```diff
@@ -1,55 +1,55 @@
-import addNewPlayer from './modules/add-players.js';
+import axios from 'axios';
 
 import './style.css';
 
-const players = [
-  {
-    name: 'Wilhelm',
-    score: 20,
-  },
-  {
-    name: 'David',
-    score: 15,
-  },
-  {
-    name: 'Willem',
-    score: 10,
-  },
-  {
-    name: 'Peter',
-    score: 15,
-  },
-  {
-    name: 'Luke',
-    score: 5,
-  },
-];
+let gameId = 'Ny7Ttc30wM2KhPKVHTkh';
+
+axios.post('https://us-central1-js-capstone-backend.cloudfunctions.net/api/games', {
+  name: 'My Awesome Game',
+}).then((response) => {
+  gameId = response.data.result;
+});
 
 const displayPlayers = (players) => {
   const playerList = document.getElementById('player-list');
   playerList.innerHTML = '';
   players.forEach((playerListItem) => {
     const playerItem = document.createElement('li');
     playerItem.className = 'player-item';
-    playerItem.innerHTML = `Name: ${playerListItem.score}`;
+    playerItem.innerHTML = `${playerListItem.user}: ${playerListItem.score}`;
     playerList.appendChild(playerItem);
   });
 };
 
 window.addEventListener('load', () => {
-  displayPlayers(players);
   const refreshBtn = document.getElementById('refresh-btn');
-  refreshBtn.addEventListener('click', () => {
-    displayPlayers(players);
+  refreshBtn.addEventListener('click', async () => {
+    try {
+      const response = await axios.get(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`);
+      const scores = response.data.result;
+      displayPlayers(scores);
+    } catch (error) {
+      // console.error(error);
+    }
   });
+
   const submitBtn = document.getElementById('submit-btn');
-  submitBtn.addEventListener('click', (e) => {
+  submitBtn.addEventListener('click', async (e) => {
     e.preventDefault();
     const inputName = document.getElementById('input-name');
     const inputScore = document.getElementById('input-score');
-    addNewPlayer(inputName.value, inputScore.value, players);
-    inputName.value = '';
-    inputScore.value = '';
-    displayPlayers(players);
+    const newName = inputName.value;
+    const newScore = Number(inputScore.value);
+    try {
+      await axios.post(`https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/${gameId}/scores/`, {
+        user: newName,
+        score: newScore,
+      });
+      inputName.value = '';
+      inputScore.value = '';
+      refreshBtn.click();
```

Comment:
- [ ] KIndly remove the` refreshBtn` function you added after making the post request. Please note that when the post request is made, it shouldn't also fetch the data and display it on the UI. The data should only be displayed when you click on the refresh button 👍🏽